theme/n21/views/dataprivacy/privacy.hbs

```diff
@@ -0,0 +1,301 @@
+	</span></span><span style='font-size:12.0pt;color:#373A3C'>Gruppenmitgliedschaften,
+	z.B. Klasse und Kurse</span></p>
+	<p class=MsoNormal style='line-height:normal;margin-bottom:0.3rem;'><span style='font-size:10.0pt;font-family:Symbol;
+	color:#373A3C'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+	</span></span><span style='font-size:12.0pt;color:#373A3C'>Geburtsdatum</span></p>
+	<p class=MsoNormal style='line-height:normal;margin-bottom:0.3rem;'><span style='font-size:10.0pt;font-family:Symbol;
+	color:#373A3C'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+	</span></span><span style='font-size:12.0pt;color:#373A3C'>Persönliche
+	Einstellungen</span></p>
+	<p class=MsoNormal style='line-height:normal;margin-bottom:0.3rem;'><span style='font-size:10.0pt;font-family:Symbol;
+	color:#373A3C'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+	</span></span><span style='font-size:12.0pt;color:#373A3C'>Zeitstempel, z.B.
+	Erstellung des Nutzer-Accounts, letzter Login</span></p>
+	<p class=MsoNormal style='line-height:normal;margin-bottom:0.3rem;'><span style='font-size:10.0pt;font-family:Symbol;
+	color:#373A3C'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+	</span></span><span style='font-size:12.0pt;color:#373A3C'>Selbst hochgeladene
+	und freigegebene Dateien, z.B. Dokumente</span></p>
+	<p class=MsoNormal style='line-height:normal;margin-bottom:0.3rem;'><span style='font-size:10.0pt;font-family:Symbol;
+	color:#373A3C'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+	</span></span><span style='font-size:12.0pt;color:#373A3C'>Selbst angelegte und
+	freigegebene Termine, Gruppentermine</span></p>
+	<p class=MsoNormal style='line-height:normal;margin-bottom:0.3rem;'><span style='font-size:10.0pt;font-family:Symbol;
+	color:#373A3C'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+	</span></span><span style='font-size:12.0pt;color:#373A3C'>Aufgabenstellungen
+	und deren Bearbeitungen mit Abgabestatus und Feedback</span></p>
+	<p class=MsoNormal style='line-height:normal;margin-bottom:0.3rem;'><span style='font-size:10.0pt;font-family:Symbol;
+	color:#373A3C'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+	</span></span><span style='font-size:12.0pt;color:#373A3C'>Zugriffe auf
+	Lerninhalte, Nutzungsdaten der jeweiligen Inhalte und übermitteltes Feedback zu
+	einzelnen Inhalten</span></p>
+	<p class=MsoNormal style='line-height:normal;margin-bottom:0.3rem;'><span style='font-size:10.0pt;font-family:Symbol;
+	color:#373A3C'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+	</span></span><span style='font-size:12.0pt;color:#373A3C'>Erstellte
```

Comment:
cleanup that messy styling

Reply:
All that messy stuff will be replaced later with a reworked data privacy text, but this is still in progress.

Reply:
n21 data privacy fully reworked.